tests/compute/subresources/pausing.go

```diff
@@ -265,7 +266,7 @@ var _ = compute.SIGDescribe("[rfe_id:3064][crit:medium][vendor:cnv-qe@redhat.com
 			Eventually(matcher.ThisVMI(newVMI), 30*time.Second, time.Second).Should(matcher.HaveConditionMissingOrFalse(v1.VirtualMachineInstancePaused))
 		})
 
-		It("[test_id:3083]should connect to serial console", func() {
+		It("[test_id:3083]should connect to serial console", decorators.Conformance, func() {
```

Comment:
Leaving comment here but it generally hold for the whole commit. 
1. Few of tests are written in a way that assume they are cheap and it is better to write one more test for one more assert rather than writing the assert to existing test.
2. Console and VNC should have separate tests/files